README.md

```diff
@@ -110,16 +81,19 @@ Lodash template function does not provide `include` method of [ejs module](http:
 
 As a result, `renderedHtml` becomes a string `<h1><a href="http://example.com">Example</a></h1>`.
 
-
-
 ## Release History
-* 1.0.0 - Changed export format to ESM
+* 1.0.0 - Initial fork of [ejs-loader](https://github.com/difelice/ejs-loader) with support for ECMA Modules and removing CommonsJS
```

Comment:
Do not recommend. Use `0.4.0-esm` or something similar. Beta versions (`<1.y.z`) are allowed to break stuff in minor releases (`0.x`). The package maintainers should decide when the thing goes to `1.0.0`.

Reply:
@al-the-x I'm good with that. I wasn't sure how to version a non version-able release 😄 . The `0.4.0-esm` is just to announce on our end where the fork begins? FWIW these versions will not be what is dictated by the original package maintainers and have no bearing or effect on the actual PRs to the original repo.

Reply:
The original package is no longer under active development and has no owner, so I'm going to venture that it will never hit `1.0.0`. I'm also not optimistic about @AtofStryker's PRs being accepted. Until that happens, this is our fork and we _are_ the maintainers.

I made the change to `1.0.0` because the special allowance for beta versions forces us to pin the package or risk grove core breaking spontaneously, so we gain none of the benefits of semantic versioning. At `1.0.0`, we have the capability to release patches that will be pulled in automatically, while reserving breaking changes for major versions.